.github/workflows/link-check-deploy.yml

```diff
@@ -0,0 +1,48 @@
+name: Link Check Deploy
+on: deployment_status
+
+jobs:
+  run:
+    name: Check
+    runs-on: ubuntu-latest
+    if: github.event.deployment_status.state == 'success'
+
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - uses: LouisBrunner/checks-action@v0.1.0
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
+        name: Check
+        status: in_progress
+        conclusion: neutral
+        output: >-
+          {"summary": "Running master diff Link Check on ${{ github.event.deployment.payload.web_url }}"}
+
+    - name: Run the link check script
+      id: check
+      uses: "iterative/link-check.action@master"
+      with:
+        configFile: "config/link-check/config.yml"
+        rootURL: "${{ github.event.deployment.payload.web_url }}"
+        output: checkAction
```

Comment:
could you clarify what actually do we output here? some json structure? what does it contain?

Reply:
also, re the exitCode - why can't we make it always return something non zero on a failure?

Reply:
There's no reason we can't, I just figured it'd be better to have the option than not.

Reply:
> could you clarify what actually do we output here? some json structure? what does it contain?

It is a JSON structure, as that's what the Check Action takes for input on its output.
name: "build-test"
on: [push]

See this slice of its README, particularly `output`

It takes a required `summary`, which is used as the headline, and a markdown `text_description` to include details, like the report of failed links for us.

```
    - uses: LouisBrunner/checks-action@v0.1.0
      if: always()
      with:
        token: ${{ secrets.GITHUB_TOKEN }}
        name: Test XYZ
        conclusion: ${{ job.status }}
        output: |
          {"summary":${{ steps.test.outputs.summary }}}
```